src/erasure.rs

```diff
@@ -587,14 +544,16 @@ pub fn recover(id: &Pubkey, window: &mut [WindowSlot], start_idx: u64, start: us
             locks[n].data()[0]
         );
     }
-    assert!(!corrupt, " {} ", id);
 
-    Ok(())
+    assert!(!corrupt, " {} ", id);
```

Comment:
we have an opportunity to do something about this, now.  if we end up with a corrupt blob, let's not put it in the ledger/window?